src/PubSub/PublishedMessage.cs

```diff
@@ -17,61 +14,35 @@ namespace PeerTalk.PubSub
     ///   TODO: Sender should really be called Author.
     ///   
     /// </remarks>
-    [ProtoContract]
-    public class PublishedMessage : IPublishedMessage
+    public partial class PublishedMessage : IPublishedMessage
     {
-        string messageId;
-
+        private Peer _sender;
```

Comment:
I hate prefixing local variables with "_".  Please remove.

Reply:
Sure.  just FYI, I was doing this because it follows dotnet corefx coding guideline https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md